presto-main/src/test/java/com/facebook/presto/cost/TestStatsCalculator.java

```diff
@@ -27,25 +27,23 @@
 import static com.facebook.presto.sql.planner.assertions.PlanMatchPattern.node;
 import static com.facebook.presto.testing.TestingSession.testSessionBuilder;
 
-public class TestCoefficientBasedStatsCalculator
+public class TestStatsCalculator
```

Comment:
The rename belongs to `Allow using ComposableStatsCalculator instead of CoefficientBasedStatsCalculator` commit, which introduces the flag to select calculator. Since the flag is used in this test, it is no longer tied to just `CoefficientBasedStatsCalculator`